projectional/src/main/java/jetbrains/jetpad/projectional/cell/ProjectionalRoleSynchronizer.java

```diff
@@ -28,6 +28,12 @@
   void setCompletion(RoleCompletion<? super ContextT, SourceT> completion);
   void setDeleteHandler(DeleteHandler handler);
   void setClipboardParameters(ContentKind<SourceT> kind, Function<SourceT, SourceT> cloner);
+  <ContentT> void supportContentKind(ContentKind<ContentT> kind,
+      Function<ContentT, SourceT> fromKind,
+      Function<SourceT, ContentT> toKind);
+  <ContentT> void supportContentKindCompatibleToItemsList(ContentKind<ContentT> kind,
```

Comment:
You have similar methods but very different names. One is simple, the other has suddenly "CompatibleToItemsList" prefix. I think it's too long and confusing. We can't write very long sentences in variable names and be extremely precise.